pkg/resource/predicates.go

```diff
@@ -135,3 +135,14 @@ func IsPropagated() PredicateFn {
 		return nn.Namespace != "" && nn.Name != ""
 	}
 }
+
+// IsNamed accepts objects that is named as the given name
```

Comment:
nitpick: :upside_down_face: 
```suggestion
// IsNamed accepts objects that is named as the given name.
```

Reply:
fixed :)